src/index.html

```diff
@@ -11,6 +11,66 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      data-qa="big"
+      class="large_sb_form large_sb_container"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="get"
+    >
+      <button
+        class="large_sb_button large_sb_content"
+        type="submit"
+        name="large_sb_search"
+      >
+        <label for="large_serch_img">
+          <img
+            class="large_sb_img"
+            src="images/Search.svg"
+            alt="magnifying_glass_large"
+            id="large_serch_img"
+          >
+        </label>
+      </button>
+      <label for="large_input_label">
+        <input
+          data-qa="keypress"
+          class="large_sb_input large_sb_content"
+          type="search"
+          placeholder="Try &quot;Los Angeles&quot;"
+          id="large_input_label"
+        >
+      </label>
+    </form>
+
+    <form
+      data-qa="small"
+      class="small_sb_form small_sb_container"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="get"
+    >
+      <button
+        class="small_sb_button small_sb_content"
+        type="submit"
+        name="small_sb_search"
+      >
+        <label for="small_serch_img">
+          <img
+            class="small_sb_img"
+            src="images/Search.svg"
+            alt="magnifying_glass_small"
+            id="small_serch_img"
+          >
+        </label>
+      </button>
+      <label for="small_input_label">
+        <input
+          class="small_sb_input small_sb_content"
+          type="search"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+          id="small_input_label"
+        >
+      </label>
+    </form>
```

Comment:
в ідеалі має виглядати так:
1) задати класи, які краще описують що саме це за елемент
2) використати загальний клас, де будуть загальні стилі та модифікатор, який буде вносити зміни до базового
3) обгорнувши інпут в лейбл - не потрібно задавати їм for/id
```suggestion
    <form
      data-qa="small"
      class="form form--small"
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="get"
    >
      <label class="form__label form__label--small">
        <img
          class="form__image form__image--small"
          src="images/Search.svg"
          alt="loop"
        >
        <input
          class="form__input form__input--small"
          type="search"
          placeholder="Try &quot;Los Angeles&quot;"
          data-qa="keypress"
        >
      </label>
    </form>
```